### PR TITLE
Ready for coding

### PR DESCRIPTION
**Pull request by clinton56**
Added testNG, Mockito, travis CI.
Set java 8 in source/target and added encoidng to maven pom.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# BattleShips game
```
Comment (KPP13) [inline README.MD]: TODO: write something about our app (at least 5 or more sentences).
Nice to use github formatting tags.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (KPP13) [inline pom.xml]: // TODO: ask Tomek and add event driven entries

Comment (clinton56) [inline pom.xml]: Done ;)

File: README.MD (hunk 1 above)
Comment (clinton56) [inline README.MD]: Added 1 line

**Comment by maciejkocur** [comment]
ok

**Pull request merged**